msteams-platform/bots/what-are-bots.md

```diff
@@ -46,17 +46,24 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 <!--- TBD: For quick scanning, see if the above information can be itemized as a list.
 --->
 
-## Next step
+## Global endpoint for bots API
 
-> [!div class="nextstepaction"]
-> [Bots and SDKs](~/bots/bot-features.md)
+When creating a bot, the global routing endpoint can be enabled by setting the `cloud` parameter as `teams` in the URL. For example, the URL should be `https://smba.trafficmanager.net/teams/v3/conversations/12345`.
+
+> [!NOTE]
+> For GCC, GCCH, and DOD tenants the bot needs to call the endpoint specific for the environment (For example, call `https://smba.infra.gcc.teams.microsoft.com` for GCC environment). If the public endpoint is called instead, the requests won't succeed and a 404 status code is returned to the bot.
```

Comment:
> call the endpoint specific for the environment

This gives 1 example (for GCC). Please link to the doc page that describes the endpoints are for all the environments.